### DISCUSSION TITLE
请问行数怎么保证一致？

### DISCUSSION THREAD

**Discussion by escwxyz** [reactjs/zh-hans.react.dev#1119] [Q&A] [answered]
我看到 prettier 忽略掉翻译文件，那不用回车的话怎能保证文件内每行长度以及行数？

**Comment by awxiaoxian2020** [comment, marked as answer]
没有明白您的问题。行数一致是在内容上显示的行数保持一致即可。如：
https://github.com/reactjs/zh-hans.react.dev/pull/1104/commits/8e99c7dc4636057b5e7f42516e2a08949d898c2f#diff-a1ff9021e75e32291f6dce5a17b5e0de80469e73e8c547e86aed1e7cf02ca00fR88

以右边更改为例，第87行后应该增加一行空行，以保持对应。

这样 diff 出来就是对应的，便于 review
